var/spack/repos/builtin/packages/py-pyqt5/package.py

```diff
@@ -12,26 +12,41 @@ class PyPyqt5(SIPPackage):
     Windows, OS X, Linux, iOS and Android. PyQt5 supports Qt v5."""
 
     homepage = "https://www.riverbankcomputing.com/software/pyqt/intro"
-    url = (
-        "https://www.riverbankcomputing.com/static/Downloads/PyQt5/5.13.0/PyQt5_gpl-5.13.0.tar.gz"
-    )
-    list_url = "https://www.riverbankcomputing.com/software/pyqt/download5"
+    url = "https://files.pythonhosted.org/packages/source/P/PyQt5/PyQt5-5.15.9.tar.gz"
+    list_url = "https://pypi.org/simple/PyQt5/"
```

Comment:
URL no longer works and older versions are no longer available for download so I deprecated them